src/test/kotlin/no/nav/familie/ef/søknad/mapper/OpplysningerOmAdresseMapperTest.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.familie.ef.søknad.mapper
+
+import no.nav.familie.ef.søknad.mapper.kontrakt.OpplysningerOmAdresseData
+import no.nav.familie.ef.søknad.mapper.kontrakt.OpplysningerOmAdresseMapper
+import no.nav.familie.ef.søknad.mock.søknadDto
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class OpplysningerOmAdresseMapperTest {
+
+    private val søknadDto = søknadDto()
+    private val opplysningerOmAdresseData =
+        OpplysningerOmAdresseData(søknadDto.søkerBorPåRegistrertAdresse, søknadDto.opplysningerOmAdresse)
+    private val dokumenter = mapOf<String, DokumentasjonWrapper>()
+
+    @Test
+    fun `mapper verdier for adresse`() {
+        val bosituasjonMapped = OpplysningerOmAdresseMapper.map(opplysningerOmAdresseData, dokumenter).verdi
+
+        assertThat(bosituasjonMapped.søkerBorPåRegistrertAdresse?.label).isEqualTo("Bor du på denne adressen?")
```

Comment:
Mindre viktig:
Litt vanskelig å se "given" verdier som vi asserter på her. Er det mulig å legge dette inn i søknaden så det blir eksplisitt. Evt lage en test to med f.eks harMeldtFlytteendring = false. (Men det blir kanskje mye å kopiere?)

Reply:
Testen er bygget opp på samme måte som du har byggt opp de andre testene, så har prøvd å forholde meg til det.
Given-verdiene kommer fra en json-fil.

[58b718a](https://github.com/navikt/familie-ef-soknad-api/pull/785/commits/58b718a1e7ef54ee9e74e6f8016c13260d41482e)